assets/dev/scss/global/_dialog.scss

```
@@ -140,6 +140,13 @@
 	}
 }
 
+#e-kit-deleted-dialog {
```

Maybe? So you know it's related to the revert?
```suggestion
#e-revert-kit-deleted-dialog {
```